ui/package.json

```
@@ -96,7 +96,8 @@
     "shebang-loader": "^0.0.1",
     "snake-case": "^3.0.3",
     "subscriptions-transport-ws": "^0.11.0",
-    "taskcluster-client-web": "^60.4.2",
+    "taskcluster-client": "68.0.4",
+    "taskcluster-client-web": "68.0.4",
```

Oh, also, this should probably be:

```suggestion
    "taskcluster-client": "^68.0.4",
    "taskcluster-client-web": "^68.0.4",
```

---

thanks! indeed